### PR TITLE
deal.II-OpenFOAM flap_perp

### PR DESCRIPTION
**Pull request by DavidSCN**
This tutorial case is the same as already existing for CalculiX. I would like to test it again and add a parallel version for the Fluid participant. Please have a look at the precice-config files, as you are there more an expert than me.

### CODE REVIEW

**Comment by DavidSCN** [comment]
Feel free to try this tutorial case already. The documentation in the precice wiki is still missing. But you only need to follow the [building instruction](https://github.com/precice/dealii-adapter/wiki/Building) and copy the executable in this directory.

**Comment by DavidSCN** [comment]
If the config files are ok (I have not tested in parallel), this should be ready to merge. The [wiki entry](https://github.com/precice/precice/wiki/Tutorial-for-FSI-with-deal.II-and-OpenFOAM) will also be completed today.

File: FSI/flap_perp/deal.II-OpenFOAM/Fluid/precice-adapter-config.yml [hunk 1]
```diff
@@ -8,12 +8,10 @@ interfaces:
   patches:
   - flap
   write-data: Force
-  provideMeshConnectivity: false
```
[the remark this answers is not in the document]
Comment (MakisH) [reply FSI/flap_perp/deal.II-OpenFOAM/Fluid/precice-adapter-config.yml]: This is part of a larger discussion, partially related to https://github.com/precice/tutorials/issues/28

File: FSI/flap_perp/deal.II-OpenFOAM/Solid/parameters.prm [hunk 2]
```diff
@@ -49,10 +49,10 @@ subsection precice configuration
     # Name of the participant in the precice-config.xml file
     set Participant         = Solid
 	
-    # Name of the node based coupling mesh in the precice-config.xml file
+    # Name of the node-based coupling mesh in the precice-config.xml file
     set Node mesh name      = Solid_nodes
 	
-    # Name of the face based coupling mesh in the precice-config.xml file
+    # Name of the face-based coupling mesh in the precice-config.xml file
```
[the remark this answers is not in the document]
Comment (MakisH) [reply FSI/flap_perp/deal.II-OpenFOAM/Solid/parameters.prm]: It is just a comment and I also have changed it in the wiki, but please feel free to change it anywhere else eventually.

**Comment by MakisH** [comment]
- [ ] add warning for pimpleFoam vs pimpleDyMFoam (see https://github.com/precice/tutorials/commit/e5f9a892587db421798c314054442605775a3fa0)

File: CHT/flow-over-plate/buoyantPimpleFoam-nutils/Allclean [hunk 3]
```diff
@@ -17,30 +17,31 @@ cd ${Participant1}
     touch ${Participant1}.foam
 
     # Remove the log files
-    rm -f ${Participant1}_blockMesh.log
-    rm -f ${Participant1}_checkMesh.log
-    rm -f ${Participant1}_decomposePar.log
-    rm -f ${Participant1}.log
-    rm -f ${Participant1}_reconstructPar.log
-    rm -f \
+    rm -fv ${Participant1}_blockMesh.log
```
Comment (MakisH) [inline CHT/flow-over-plate/buoyantPimpleFoam-nutils/Allclean]: These files have already been fixed in another commit, already into master. Please revert these changes, merge from master, and keep this PR only about files into `FSI/flap_perp/OpenFOAM-deal.II`.

Comment (DavidSCN) [reply CHT/flow-over-plate/buoyantPimpleFoam-nutils/Allclean]: This changes have been committed in db3e882 and have nothing to do with the recent commits. Or should this be the instruction for merging?
